CHANGELOG.md

```diff
@@ -12,7 +10,11 @@
 
 ### Fixed
 
-### Dependency updates
+## [0.2.23] 2020-03-10
```

Comment:
```suggestion
## [0.2.24] 2020-03-10
```

Reply:
Good catch 😄 The easier the task, the easier errors slip in